coalib/settings/ConfigurationGathering.py

```diff
@@ -283,6 +289,13 @@ def load_configuration(arg_list,
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    for section_name in exclude_sections:
+        if section_name in sections:
+            del sections[section_name]
+        else:
+            logging.warning('The {} section doesn\'t exist'
+                            .format(section_name))
```

Comment:
1. Add a period
2. Please add a bit more context, like "the section to exclude doesn't exist, ignoring it" or so (so where was it given by the user, what's the case and what's gonna happen :+1:)